act-rules-format.md

```diff
@@ -206,6 +208,62 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+### Mapping Associated/Affiliated/Secondary Requirements ###
+
+An ACT Rule <em class="rfc2119">may</em> show that the outcomes of the rule have secondary [=accessibility requirements=] for that test subject. When all of the outcomes for a test target in failed test cases is `failed` but an accessibility requirement cannot be listed as <dfn>not satisfied</dfn>, the accessibiity requirement must be listed as a secondary requirement and <dfn>further testing is needed</dfn>. When all outcomes for a test target in passed test cases are not `passed`, the mapped secondary accessibility requirement is <dfn>further testing is needed</dfn>.
```

Comment:
```suggestion
An ACT Rule <em class="rfc2119">may</em> show that the outcomes of the rule have secondary [=accessibility requirements=] for that test subject. When one or more of the outcomes for a test target is `failed` but an accessibility requirement cannot be listed as <dfn>not satisfied</dfn> because <dfn>further testing is needed</dfn>, the accessibility requirement must be listed as a secondary requirement. When one or more of the outcomes is `passed` , the mapped secondary requirement can be <dfn>satisfied</dfn>, <dfn>further testing is needed</dfn>, or <dfn>not satisfied</dfn>.
```